docs/application-connector/08-05-trigger-lambda-with-event.md

```diff
@@ -64,35 +75,35 @@ This guide shows how to create a simple lambda function and trigger it with an e
    }
    ```
 
-2. Get the `externalName` of the Service Class of the registered service.
+3. Expose the `externalName` of the Service Class of the registered service.
```

Comment:
```suggestion
3. Expose `externalName` of the Service Class of the registered service.
```

Reply:
I'd leave it as it is as in this particular case, the `externalName` doesn't just function as a variable name, but is also an integral part of the sentence.